tests/integ/modin/frame/test_align.py

```diff
@@ -17,33 +17,42 @@
 from tests.integ.utils.sql_counter import SqlCounter, sql_count_checker
 
 
+def eval_dataframe_align_result(
```

Comment:
Thanks for refactoring this @sfc-gh-yzou! Can this test modification and tests for `on_row_position_columns` be added to the align series tests as well (series/test_align.py), or refactor in another PR?

Reply:
added a refactor for series also.